task/buildah/0.1/README.md

```diff
@@ -0,0 +1,28 @@
+# buildah task
+
+## Description:
+Buildah Task builds source into a container image using Project Atomic's Buildah build tool.It uses Buildah's support for building from Dockerfiles, using its buildah bud command.This command executes the directives in the Dockerfile to assemble a container image, then pushes that image to a container registry.
```

Comment:
```suggestion
Buildah Task builds source into a container image using Project Atomic's Buildah build tool. It uses Buildah's support for building from Dockerfiles, using its buildah bud command. This command executes the directives in the Dockerfile to assemble a container image, then pushes that image to a container registry.
```

Reply:
removed this task in prior update so resolving comments re: buildah